### ISSUE TITLE
Calendar Page

### ISSUE THREAD

**Issue by andrewnl** [gitumarkk/newlife-github#2]
> So what is the last thing left before it is pushed live? Also wanted to add links to each milestone page, it will be quick but do you think it's necessary?
> Also are you happy with the styling or should I update it?
- Yes, add the links to the milestone pages to each date in the left column. I'm just going to update this week's schedule and we'll see how it looks.

**Comment by andrewnl** [comment]
Here are some notes comparing http://nlife.ca/teamtest/calendar.html with http://nlife.ca/schedule
1. Column ordering: need to be the same order as http://nlife.ca/schedule (which is the meeting timeline)
2. Parsing to get people: You are right that seaching for ' - ' is a good strategy. (Note the space each side of the '-'). Something went wrong with Food on Dec 13
3. Date column (first col). Remove the year so the date does not wrap
4. Link date to milestone on github
5. Overview column must be treated differently. You get it right on http://nlife.ca/teamtest/index.html

Those are the only issues I can spot for now.

**Comment by gitumarkk** [comment]
@andrewnl Made the changes and committed to `master`.

**Comment by andrewnl** [comment]
Sorry for the delay in checking it.
I pushed it to the website but we seem to get a "white screen of death".
http://nlife.ca/teamtest/calendar.html

**Comment by gitumarkk** [comment]
@andrewnl I moved everything to index.html to allow for linking the href labels to the more detailed view, calendar.html should no longer be used.

**Comment by andrewnl** [comment]
Ok, I see.
Where do I find the view of the Sunday, with all the pictures? (the old index.html?)

**Comment by gitumarkk** [comment]
click on any of the table headings and it should take you to the granular view.

**Comment by andrewnl** [comment]
Yes, that's great, but that only gives me the view for one team.
Where do I find the overview of one Sunday that we had before, with all the pictures? (the old index.html?)

**Comment by gitumarkk** [comment]
@andrewnl in the detail view there is a navbar with 3 tabs, `calendar`, `week` and `duties`, select `week` to get the `pictures/overview`. I was working on the assumption that the `calendar view` was the main view and the `pictures/overview` would be linked to the `calendar view`.

**Comment by andrewnl** [comment]
Ah, I see it now. Great!
